components/org.wso2.carbon.identity.conditional.auth.functions.http/src/test/java/org/wso2/carbon/identity/conditional/auth/functions/http/HTTPGetFunctionImplTest.java

```diff
@@ -475,4 +506,32 @@ public Map<String, String> dummyTokenEndpoint(@HeaderParam("Authorization") Stri
         }
         return response;
     }
+
+    @FunctionalInterface
```

Comment:
Add comments explaining the use of the interface and its implementation

Reply:
Addressed with 46c8e19185f03a9c7a8e20a87f85cae4e5e0ea96